app/src/main/java/org/oppia/app/player/state/StateFragmentPresenter.kt

```diff
@@ -94,10 +117,35 @@ class StateFragmentPresenter @Inject constructor(
 
   private fun subscribeToCurrentState() {
     ephemeralStateLiveData.observe(fragment, Observer<EphemeralState> { result ->
+      if (result.state.interaction.id.equals("NumericInput"))
+        addNumberInputContentCard(result.state.interaction.customizationArgsMap.get("placeholder")!!.normalizedString)
```

Comment:
Added `{}` to this if statement to make it more readable.

Reply:
okay